### ISSUE TITLE
Missing freeradius plugin package for Ubuntu 22.04 LTS

### ISSUE THREAD

**Issue by jonjacs** [privacyidea/privacyidea#3266]
Will there be a package built for Ubuntu 22.04 LTS?

```
# apt search privacyidea-radius
Sorting... Done
Full Text Search... Done
#
```

**Comment by plettich** [comment]
There is a package for Ubuntu 22.04 in the community devel-repository:
`deb http://lancelot.netknights.it/community/jammy/devel jammy main`
There You can find the other privacyidea packages as well.
We haven't released a stable repository for Ubuntu 22.04 yet.

**Comment by jonjacs** [comment]
Excellent! Thank you!
